### PR TITLE
added blueprint and blueprint details page inside page directory

### PR DESCRIPTION
**Pull request by mageexcel**

### CODE REVIEW

**Comment by mageexcel** [comment]
@Skraye , Can you please tell me what this is for?
image

**Comment by tchiotludo** [comment]
> @Skraye , Can you please tell me what this is for? image

It's the pagination size, you can skip for now

**Comment by mageexcel** [comment]
Should I add this static for now
image

**Comment by tchiotludo** [comment]
> Should I add this static for now image

It should be dynamic and feed from the api

**Comment by mageexcel** [comment]
> It should be dynamic and feed from the api

But, I don't have the api right now so, can you please tell me how I should make it dynamic as I don't know in which format data will come from the api.

**Comment by Skraye** [comment]
I'll send you an API example in chat and some details!

**Comment by mageexcel** [comment]
> I'll send you an API example in chat and some details!

Okay @Skraye thanks

File: components/blueprints/About.vue [hunk 1]
```diff
@@ -0,0 +1,46 @@
+<template>
+<div class="mt-5">
+    <h3>About this blueprint</h3>
+    <div class="title">
+        <p>{{ title }}</p>
+    </div>
+    <div class="description">{{ description }}</div>
```
Comment (Skraye) [inline components/blueprints/About.vue:7]: This has to be displayed has markdown

**Review by Skraye — CHANGES_REQUESTED** [review]
Be careful about responsiveness :
image

**Comment by tchiotludo** [comment]
close in favor of https://github.com/kestra-io/docs/pull/557

**Pull request closed without merging**